test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get users_show_url
```

Comment:
when you run `rails test` in your project on your computer, this test would fail - I would remove it for now and we can handle tests within this task: https://trello.com/c/rZboBjEy/4-add-rspec

Reply:
OK, i tried it and it failed as you said. Will remove it.